backend/daemon/api/accounts/v1alpha/accounts.go

```diff
@@ -27,7 +27,22 @@ type Server struct {
 }
 
 // NewServer creates a new Server.
-func NewServer(id *future.ReadOnly[core.Identity], blobs *hyper.Storage) *Server {
+func NewServer(ctx context.Context, id *future.ReadOnly[core.Identity], blobs *hyper.Storage) *Server {
+	go func() {
```

Comment:
Can you set trust on our own account when the initial registration happens? For existing accounts we can do that with a migration.

Starting goroutines in a "constructor" functions, and passing `context.Context` to such functions is a pattern I've learned to be very cautious about. I'd like to avoid doing that if possible.

Reply:
Sure, I thought about it but I did not know what our accountID was when migration. How do I know in the migration script which public key id is the one corresponding with our own account? the one with index 1?

Reply:
Sure, at the beginning I thought I was doing that, but Then I realized that I did not know what my own account was in the migration script. How would I know? Is the one with index 1? @burdiyan ?

Reply:
Changed to index 1 at migration time

Reply:
I guess it's safe to assume that our account is with ID 1. BUT, I'd be very very careful with that, because during vacuum rowids could change (although I think they don't when they are aliased to INTEGER PRIMARY KEY columns).

The safest bet is to read it from the file, there's `maybeLoadAccountKey` function that you can take a look at.